Tasks/ExtractFilesV1/extractfilestask.ts

```diff
@@ -10,6 +10,9 @@ var cleanDestinationFolder: boolean = tl.getBoolInput('cleanDestinationFolder',
 var repoRoot: string = tl.getVariable('System.DefaultWorkingDirectory');
 tl.debug('repoRoot: ' + repoRoot);
 
+var overwriteDestinationVar: string = tl.getVariable('ExtractFile.OverwriteDestination');
```

Comment:
Let's call this something like `EXTRACT_FILES_PREVENT_FILE_OVERWRITE`. The naming you are using is for another type of variable.